components/blocks/BlockComponentName.vue

```diff
@@ -0,0 +1,14 @@
+<script setup lang="ts">
+type Block = {
+	id: string;
+};
```

Comment:
Use interfaces for object structure typing

```suggestion
interface Block {
	id: string;
}
```